demisto_sdk/__main__.py

```diff
@@ -417,25 +421,51 @@ def prepare_content(ctx, **kwargs):
     """
     This command is used to prepare the content to be used in the platform.
     """
-    if click.get_current_context().info_name == "unify":
-        kwargs["unify_only"] = True
-
-    check_configuration_file("unify", kwargs)
-    # Input is of type Path.
-    kwargs["input"] = str(kwargs["input"])
-    file_type = find_type(kwargs["input"])
-    if marketplace := kwargs.get("marketplace"):
-        os.environ[ENV_DEMISTO_SDK_MARKETPLACE] = marketplace.lower()
-    if file_type == FileType.GENERIC_MODULE:
-        from demisto_sdk.commands.prepare_content.generic_module_unifier import (
-            GenericModuleUnifier,
+    assert any(
+        [kwargs["all"], kwargs["input"]]
+    ), "Either '-a' or '-i' parameters must be provided."
+    assert not all(
+        [kwargs["all"], kwargs["input"]]
+    ), "Only one of '-a' and '-i' parameters should be provided."
```

Comment:
we can `assert sum(kwargs["all"], kwargs["input"]), "either all or input must be provided"`, a bit cleaner :)

Reply:
-i is a string and not a boolean so it can't be summed :(
Unless converting it first to a boolean which won't look nice again. Also, would need to check that sum == 1 as we don't want to allow both parameters.

Reply:
`sum(bool(kwargs["all"])),bool(kwargs["input"]))) == 1` isn't too ugly, and is less code, so I recommend but won't insist 🥲

Reply:
no problem, updated